purchase_order_uninvoiced_amount/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Tecnativa - Manuel Calero
+# Copyright 2020 Tecnativa - João Marques
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Purchase Order Univoiced Amount",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "version": "18.0.1.0.1",
```

Comment:
Bump module version to `18.0.1.0.0` according to https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-18.0
```suggestion
    "version": "18.0.1.0.0",
```

Reply:
ups, that was a mistake